app/services/rapidfire/base_service.rb

```diff
@@ -1,6 +1,6 @@
 module Rapidfire
   class BaseService
-    if Rails::VERSION::MAJOR == 4 || Rails::VERSION::MAJOR == 5
+    if Rails::VERSION::MAJOR >= 4
       include ActiveModel::Model
     else
```

Comment:
@aksh93 If we are removing support for versions < 4, why not remove the else wherever we are checking versions >=4 ?

Reply:
Yeah makes sense